CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 4.6.0 / 2023-07-26
+
+### ✨ New Features
+
+- Add command to really delete data in tracimcli
+- Increase size of emojis in TinyMCE
```

Comment:
This is a really new feature or a bug fix ?

Reply:
It's not really a bug fix because it wasn't a bug, it's just an ux change